app/models/conference.rb

```diff
@@ -113,6 +113,24 @@ def confirmed
   before_create :create_email_settings
   before_create :add_color
 
+  def date_range_string
+    startstr = 'Unknown - '
+    endstr = 'Unknown'
+    if start_date.month == end_date.month && start_date.year == end_date.year
+      startstr = start_date.strftime('%B %d - ')
+      endstr = end_date.strftime('%d, %Y')
+    elsif start_date.month != end_date.month && start_date.year == end_date.year
+      startstr = start_date.strftime('%B %d - ')
+      endstr = end_date.strftime('%B %d, %Y')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.